core/src/main/kotlin/pages/ContentNodes.kt

```diff
@@ -101,7 +101,8 @@ data class ContentList(
     override val dci: DCI,
     override val platforms: Set<PlatformData>,
     override val style: Set<Style>,
-    override val extras: Set<Extra>
+    override val extras: Set<Extra>,
+    val start: Int = 1
```

Comment:
This only makes sense when the list is ordered. It should either be an `extra` or we should create separate classes for ordered and unordered lists. I'm slightly leaning towards the first solution, @Kordyjan ?

Reply:
I haven't thought about reusing extras. I guess I have mistakenly taken `Extra` to be enum class so it couldn't store any value.